composer.json

```diff
@@ -13,7 +13,7 @@
         "irc": "irc://irc.freenode.net/phergie"
     },
     "require": {
-        "php": "^5.5||^7.0",
+        "php": "^5.6||^7.0",
```

Comment:
My personal preference is `^7.0 || ^5.6` :smile: . Mainly because it ensure we don't run into any unintended side effects when PHP 8 comes out